install/init/crds-v1.yaml

```diff
@@ -0,0 +1,1024 @@
+---
```

Comment:
There should be a corresponding change to the installation introduction document